example_sock_elf_test.go

```diff
@@ -77,8 +77,6 @@ var program = [...]byte{
 	0010, 0000, 0000, 0000, 0000, 0000, 0000, 0000, 0030, 0000, 0000, 0000, 0000, 0000, 0000, 0000,
 }
 
-const sockexPin = "/sys/fs/bpf/sockex1"
```

Comment:
This change looks unrelated. Maybe move into a separate commit?

Reply:
I'v removed it from this PR.